.golangci.yml

```diff
@@ -4,13 +4,90 @@ run:
 #   timeout: 5m
 
 linters:
+  # Disable all linters.
+  # Default: false
   disable-all: true
+  # Enable specific linter
+  # https://golangci-lint.run/usage/linters/#enabled-by-default-linters
   enable:
+    - asciicheck
+    - bidichk
+    - bodyclose
+    - contextcheck
+    - cyclop
+    - deadcode
+    - depguard
+    - dogsled
+    - dupl
+    - durationcheck
     - errcheck
+    - errname
+    - errorlint
+    - exhaustive
+#    - exhaustivestruct <- disabled because we don't want to use every element of every struct
+    - exportloopref
+    - forbidigo
+    - forcetypeassert
+    - funlen
+    - gci
+#    - gochecknoglobals <- disabled because we want some globals
+    - gochecknoinits
+    - gocognit
+    - goconst
+    - gocritic
+    - gocyclo
+    - godot
+#    - godox <- detects TODO/BUG/FIXME and we may wantnthis later but isn't appropriate now
+#    - goerr113 <- disabled due to lack of comprehension
     - gofmt
-    # - goimports
-    # - golint
-    - maligned
+    - gofumpt
+    - goheader
+    - goimports
+    - gomoddirectives
+    - gomodguard
+    - goprintffuncname
+    - gosec
+    - gosimple
+    - govet
+#   - ifshort <- seems to be of questionable value
+    - importas
+    - ineffassign
+#    - ireturn <- disabled because we want to return interfaces
+#    - lll <- disabled as it is a bit much.  Maybe we have a desired limit?
+    - makezero
+    - misspell
+    - nakedret
+    - nestif
+    - nilerr
+    - nilnil
+#    - nlreturn <- disabled as it doesn't auto-fix something simple and doesn't add anything useful
+    - noctx
+#    - nolintlint <- disabled because we use nolint in some places
+    - paralleltest
```

Comment:
https://github.com/kunwardeep/paralleltest

I don't think I understand what this one does o_O

Reply:
maybe encourages you to write tests that are parallelized?

Reply:
if so I wasn't really aware of the feature before -- cut it?

Reply:
yeah SGTM